.env

```diff
@@ -46,6 +46,7 @@ MEILI_TOKEN=dev
 
 EXPERIMENTATION_KEY=6FA4C8D3DD2400B9D614BB7A229FD69B
 #GROWTHBOOK_CLIENT_KEY=override_me_in_env.development
+#GROWTHBOOK_API_CONFIG_CLIENT_KEY=override_me_in_env.development
```

Comment:
in dev config will always be `{}`, if something is needed to test gb key can be added as with normal feature flags/exp